08- fraction_calculator/FracCalc.java

```diff
@@ -101,10 +105,36 @@ else if (Operation.equals("/")) {
       int n1  =  Integer.parseInt(N1);
```

Comment:
for example, you could do something like `int n1 = Integer.parseInt(getNumerator(fraction1));` as long as you have a method that returns the numerator.
then getting the numerator for fraction 2 would be as simple as 
`int n2 = Integer.parseInt(getNumerator(fraction2));`

Reply:
same idea with denominator and whole value